src/ui/components/shared/Forms/TextInput.tsx

```diff
@@ -1,27 +1,28 @@
 import classNames from "classnames";
 import React from "react";
 
+import styles from "./TextInput.module.css";
+
 export default React.forwardRef<
   HTMLInputElement,
   Omit<React.HTMLProps<HTMLInputElement>, "type" | "className"> & {
     textSize?: "base" | "lg" | "2xl";
     center?: boolean;
+    theme?: "light" | "dark";
     className?: string;
   }
 >(function TextInput(props, ref) {
-  const { textSize, center } = props;
+  const { textSize, center, theme, className, ...otherProps } = props;
+
+  const themeClass = theme || "default";
+  const textSizeClass = textSize ? `text-${textSize}` : "text-sm"; // Tailwind class for textSize
```

Comment:
`text-${textSize}` may break with tailwind pruning. If we use tailwind, we need to explicitly include all the tailwind strings in the source to avoid unexpected pruning.